src/interfaces/src/plugin/data/interfaces/interface_state.c

```diff
@@ -71,7 +71,7 @@ int interfaces_interface_state_hash_element_set_name(interfaces_interface_state_
         return (*el)->state.name == NULL;
     }
 
-    return 0;
+    return -1;
```

Comment:
Should return 0. Value for the passed `name` parameter can be `NULL`. If the name is set to NULL, data is only freed and the function should return 0 as a success error code.